pkg/logger/logger.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
```

Comment:
Is this file useful?

Reply:
Yes. As some functions does not implemented in openpitrix/logger.
I suggest use this update, and then to add the lost functions implementaion in  openpitrix/logger later.

Reply:
Which function not implemeted in openpitrix.io/logger?

Reply:
logger.NewLogger() does not  exist in  openpitrix.io/logger.

Reply:
![screenshot from 2019-03-05 00-02-55](https://user-images.githubusercontent.com/5917832/53746064-5d902b00-3edb-11e9-91ca-eb410230d664.png)

NewLogger in openpitrix.io/logger named New. 
You can remove the file, and make the change in the image.

Reply:
updated.